src/pages/products/sgid/planning/housing-unit-inventory.astro

```diff
@@ -0,0 +1,184 @@
+        some counts being assigned manually using information from recent aerial photography and other sources.
+      </li>
+      <li>
+        For cases where individual condos, townhomes, and single family homes have associated common areas, the
+        inventory aggregates and combines the geometries and attributes of the individual units and common areas into a
+        single composite parcel feature called an "owned unit grouping". This sums, averages, and counts the appropriate
+        attributes of the contributing parcels to better characterize the density of these residential land uses
+        (dwelling units per acre or DUA).
+      </li>
+      <li>
+        Every county uses a slightly different set of designations when defining residential parcel types. The inventory
+        attempts to standardize all the different classifications into the following categories:
+        <ul class="list-disc pl-8">
+          <li>Townhouse</li>
+          <li>Condo</li>
+          <li>Apartment</li>
+          <li>Single family</li>
+          <li>Single family with Accessory Dwelling Units (ADUs, currently only for Salt Lake County)</li>
+          <li>Duplex</li>
+          <li>Mixed use (commercial and residential)</li>
+          <li>Mobile home</li>
+          <li>Pud (mixed)</li>
+          <li>Group quarters</li>
+        </ul>
+      </li>
+      <li>
+        The inventory doesn't include the following types of residential properties:
+        <ul class="list-disc pl-8">
+          <li>Nursing, rehabilitation, or assisted living centers</li>
+          <li>Extended stay hotels</li>
+          <li>Incarceration facilities</li>
+          <li>Student housing or dormitories</li>
+          <li>ADUs outside Salt Lake County</li>
+        </ul>
+      </li>
+    </ul>
+    <p>The inventory uses the following fields to provide data on each parcel or owned unit grouping:</p>
+    <ul class="list-disc pl-4">
+      <li><code>TYPE</code>: General housing type, single- or multi-family</li>
+      <li><code>SUBTYPE</code>: More detailed housing type classification</li>
+      <li>
+        <code>IS_OUG</code>: 1 or Yes for features that are owned unit groupings (as opposed to a single-parcel home or
+        apartment)
+      </li>
+      <li><code>UNIT_COUNT</code>: The number of dwelling units, based on address point data</li>
+      <li><code>DUA</code>: Density in dwelling units per acre</li>
+      <li><code>ACRES</code>: Total acreage</li>
+      <li><code>TOT_BD_FT2</code>: Total building(s) square footage</li>
+      <li><code>TOT_VALUE</code>: Total market value</li>
+      <li><code>APX_BLT_YR</code>: Approximate year built</li>
+      <li><code>BLT_DECADE</code>: The decade the structure(s) were built</li>
+      <li>
+        <code>CITY</code>: The features' city based on the SGID's <a href="products/sgid/boundaries/citycountystate"
```

Comment:
This url will likely change to `boundaries/municipal` or something like that when we explode that page.
```suggestion
        <code>CITY</code>: The features' city based on the SGID's <a href="/products/sgid/boundaries/citycountystate"
```